Core/Core/Models/Plannable.swift

```diff
@@ -36,24 +36,12 @@ public final class Plannable: NSManagedObject, WriteableModel {
     @NSManaged public var canvasContextIDRaw: String?
     @NSManaged public var contextName: String?
     @NSManaged public var date: Date
+    @NSManaged public var pointsPossibleRaw: NSNumber?
+    @NSManaged public var userID: String?
 
     public var pointsPossible: Double? {
-        get {
-            willAccessValue(forKey: "pointsPossible")
-            defer { didAccessValue(forKey: "pointsPossible") }
-
-            return primitiveValue(forKey: "pointsPossible") as? Double
-        }
-        set {
-            willChangeValue(forKey: "pointsPossible")
-            defer { didChangeValue(forKey: "pointsPossible") }
-
-            guard let value = newValue else {
-                setPrimitiveValue(nil, forKey: "pointsPossible")
-                return
-            }
-            setPrimitiveValue(value, forKey: "pointsPossible")
-        }
+        get { return pointsPossibleRaw?.doubleValue }
+        set { pointsPossibleRaw = NSNumber(value: newValue) }
```

Comment:
You didn't have to change it, I just didn't recognize the other pattern. It is nice to have consistency though.

Reply:
no, this is way better, not sure why i went the other way